tests/plotting/test_plotting.py

```diff
@@ -623,9 +623,9 @@ def test_set_parallel_scale_invalid():
 def test_plot_no_active_scalars(sphere):
     plotter = pv.Plotter()
     plotter.add_mesh(sphere)
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError), pytest.warns(PyVistaDeprecationWarning):
         plotter.update_scalars(np.arange(5))
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError), pytest.warns(PyVistaDeprecationWarning):
         plotter.update_scalars(np.arange(sphere.n_faces))
```

Comment:
```suggestion
    with pytest.raises(ValueError), pytest.warns(PyVistaDeprecationWarning):
        plotter.update_scalars(np.arange(sphere.n_faces))
        if pv._version.version_info >= (0, 46):
            raise RuntimeError("Convert error this method")
        if pv._version.version_info >= (0, 47):
            raise RuntimeError("Remove this method")
```

Reply:
Ah thanks for pointing this out. I forgot that we do this sometimes. Will implement.